packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -89,13 +89,20 @@ contract TaikoL1 is EssentialContract, ITaikoL1, TaikoEvents, TaikoErrors {
             }
         }
 
+        require(_params.length == _txLists.length, "mismatched params length");
```

Comment:
I tend to think, it might be better like this below if we wanna support `calldata` efficiently. 

```
        if(txLists.length != 0) {
            require(params.length == _txLists.length, "mismatched params length"));
        }
```

So currently protocol can handle both, but with the above change, we would have to create empty `_txLists` -matched to the length of `_params`, even if we use blob OR some empty if we use blob + calldata both, separately, paralel with one L1 TXN.

I dont think it is practical to use different data avail. layer for multiple L2 blocks but one L1 TXN. (Maybe i'm wrong in here, and in this case inefficiency is i guess negligible).

cc. @Brechtpd  as it is more of a question arised from Gwyneth adaptation.